base/applications/rapps/lang/tr-TR.rc

```diff
@@ -61,14 +62,14 @@ BEGIN
     AUTOCHECKBOX "&Kurulumdan Sonra Programın Kurulum Dosyasını Sil", IDC_DEL_AFTER_INSTALL, 16, 100, 218, 12
     GROUPBOX "Yazılım kaynağı", -1, 4, 118, 240, 46
     AUTORADIOBUTTON "Varsayılanı kullan", IDC_SOURCE_DEFAULT, 15, 132, 74, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "Belirtilen kaynağı kullan", IDC_USE_SOURCE, 15, 147, 74, 10
+    AUTORADIOBUTTON "Belirtilen kaynağı kullan", IDC_USE_SOURCE, 15, 147, 100, 10
     EDITTEXT IDC_SOURCE_URL, 97, 147, 140, 12, ES_AUTOHSCROLL | WS_DISABLED
     GROUPBOX "Vekil Sunucu", -1, 4, 166, 240, 76
     AUTORADIOBUTTON "Sistem Vekil Sunucu Ayarları", IDC_PROXY_DEFAULT, 15, 180, 210, 10, WS_GROUP | WS_TABSTOP
     AUTORADIOBUTTON "Doğrudan (Vekil sunucu yok.)", IDC_NO_PROXY, 15, 195, 210, 10
     AUTORADIOBUTTON "Vekil Sunucu", IDC_USE_PROXY, 15, 210, 74, 10
     EDITTEXT IDC_PROXY_SERVER, 90, 210, 147, 12, ES_AUTOHSCROLL | WS_DISABLED
-    LTEXT "Şunun i̇çin vekil sunucu yok", -1, 26, 226, 64, 10
+    LTEXT "Şunun i̇çin\nvekil sunucu yok", -1, 26, 226, 64, 8
```

Comment:
You changed line height from 10 to 8, but it's supposed to be the same height across all translations. (and given that you added "\n" linebreak here makes it even more suspicious)

Reply:
> You changed line height from 10 to 8, but it's supposed to be the same height across all translations. (and given that you added "\n" linebreak here makes it even more suspicious)

I changed it back to 10 but I don't know if it fits or not, please give me a suggestion for this